Extensions/xkit_preferences.js

```diff
@@ -39,6 +39,29 @@ XKit.extensions.xkit_preferences = new Object({
 			'<a style="color:transparent;" class="tab_anchor" href="#" onclick="return false">XKit</a>' +
 			'</div>';
 
+		if (XKit.browser().mobile) {
+			XKit.tools.add_css('#xkit-window, #xkit-window-old { max-width: 650px !important; width: 100% !important; max-height: 100%; overflow: scroll; margin: auto 0 !important;} .xkit-window-buttons { padding-top: 0 !important; } .xkit-window-buttons .xkit-button { height: 40px !important; padding: 0 !important; }','mobile_window_fix');
+
+
+			var mobile_control_panel = '#xkit-control-panel { width: 100%; height: 100%; top: 0; left: 0; margin: 0 !important; } ' +
+				'#xkit-control-panel-inner { height: calc(100% - 40px); padding: 0; overflow: scroll; } ' +
+				'#xkit-control-panel-tabs { display: flex; overflow: scroll; whitespace: no-wrap; } ' +
+				'#xkit-control-panel-tabs div { float: none; whitespace: no-wrap; } ' +
+				'#xkit-extensions-panel-left { height: 100% !important; top: 0; left: 0; } ' +
+				'.xkit-extensions-display-type-switcher { bottom: 0; } #xkit-extensions-panel-left-search { left: 0; bottom: 0; } ' +
+				'#xkit-extensions-panel-right { height: calc(100% - 40px); width: 100%; } ' +
+				'#xkit-extensions-panel-right.xkit-wide-panel { left: 0; width: 100%; } ' +
+				'#xkit-extensions-panel-top { min-height: 100px; height: unset; } ' +
+				'#xkit-extensions-panel-top .buttons, #xkit-extension-enabled, #xkit-extension-internal-label, #xkit-extensions-panel-top .more-info, #xkit-extensions-panel-top .version, #xkit-extensions-panel-top .title { display: block; position: relative; right: unset; bottom: unset; top: unset; } ' +
+				'#xkit-extension-enabled { top: 5px; }' +
+				'.xkit-change-ext-setting-checkbox { font-size: 15px !important; bottom: 7px; top: 5px; }  .xkit-change-ext-setting-checkbox b { position: relative; bottom: -3px; } ' +
+				'.xkit-extension-setting .checkbox { height: unset !important; min-height: 30px; } ' +
+				'.xkit-extension-setting .title { font-size: 15px !important; } .xkit-extension-setting { padding: 10px 15px 10px 15px !important; } ' +
```

Comment:
.title should have `position: relative` on it to make it behave as an inline label